### PR TITLE
Fix compare sku function not found

### PR DESCRIPTION
**Pull request by vitoria**
#### What is the purpose of this pull request?
The compare sku function not found.

#### What problem is this solving?
There was an error because this method wasn't found.

#### How should this be manually tested?
https://waza--storecomponents.myvtex.com

#### Types of changes

* [X] Bug fix (a non-breaking change which fixes an issue)
* [ ] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Update props passed to `ShippingSimulator`.
 - Sku items sorted by price.
 
+#Fixed
```
Comment (brunojdo) [inline CHANGELOG.md]: `###`

**Review by brunojdo — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (vitoria) [reply CHANGELOG.md]: Done!

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Update props passed to `ShippingSimulator`.
 - Sku items sorted by price.
 
+###Fixed
```
Comment (brunojdo) [inline CHANGELOG.md]: whitespaces

**Review by brunojdo — CHANGES_REQUESTED** [review]

**Review by lucasecdb — APPROVED** [review]

**Review by andreldsa — APPROVED** [review]

**Comment by andreldsa** [comment]
I already solved that problem, on PR https://github.com/vtex-apps/product-details/pull/31

**Pull request closed without merging**